pkg/cryptoutils/publickey.go

```diff
@@ -148,8 +148,14 @@ func ValidatePubKey(pub crypto.PublicKey) error {
 	//   * Public key Q's x and y are within [0, p-1]
 	//   * Public key Q is on the curve
 	//   * Public key Q's order matches the subgroups (nQ = Ø)
-	p, err := goodkey.NewKeyPolicy(&goodkey.Config{FermatRounds: 100}, nil)
-	p.AllowECDSANISTP521 = true
+	allowedKeys := goodkey.LetsEncryptCPS()
```

Comment:
In case their definition of an allowed key ever changes, can we set each of the key fields to true? Also helps with readability.

Reply:
Like so?